mod-source-record-manager-server/src/test/java/org/folio/verticle/consumers/RawMarcChunkConsumersVerticleTest.java

```diff
@@ -279,7 +275,7 @@ public void shouldNotObserveValuesWhenJobExecutionIdNotCreated() throws Interrup
   }
 
   @Test
-  public void shouldNotSendAnyEventsForDuplicates() throws InterruptedException {
+  public void shouldSendDIErrorForDuplicatesWhenParsedRecordIsNull() throws InterruptedException {
```

Comment:
seems we need to get this test back because event should not be sent when a duplicate chunk is received